fields/layouts/home.html

```diff
@@ -34,12 +34,25 @@ <h1>
     {{- range $field.Params.subfields.content -}}
       {{- $subfield := index (where $.Site.Pages ".Params.uid" .) 0 -}}
       {{- $subfield_data := (slice) -}}
-      {{- range (first 5 $subfield.Params.courses) -}}
-          {{- $url := delimit (slice (strings.TrimSuffix "/" $staticApiBaseUrl) "/" .id "/data.json") "" -}}
-          {{- $data := getJSON $url -}}
-          {{- $data = merge $data (dict "url_path" .id) -}}
-          {{- $subfield_data = $subfield_data | append (dict .id $data) -}}
-      {{- end -}}
+      {{ with $subfield.Params.courses }}
+        {{- range (first 5 $subfield.Params.courses) -}}
+            {{- $id := .id -}}
+            {{- $url := delimit (slice (strings.TrimSuffix "/" $staticApiBaseUrl) "/" $id "/data.json") "" -}}
+            {{ with resources.GetRemote $url }}
+              {{ with .Err }}
+                {{- $errorMessage := printf "Failed to fetch sub-fields on %v via %v with error: %v" site.BaseURL $url .Err -}}
+                {{- partial "sentry_capture_message.html" $errorMessage -}}
```

Comment:
@gumaerc Not sure if this and all other partials, used inside `JS`/`script tag` will work as expected as loading this partial here will render another `script` tag inside an already loaded script.
I tried to test this but couldn't. 
I also tried to use custom/raw JS here but it also had some issues.

Let me know if you have any thoughts on this

Reply:
@MAbdurrehman12 Good catch here, I didn't notice this scenario.  I think the simplest solution would be to just create another partial, something like `sentry_capture_message_internal.html` with just the raw JS and call it explicitly.  Then, in `sentry_capture_message.html`, we can call that partial as well inside of the script tags.